app/screens/home/RouteList.tsx

```diff
@@ -721,29 +729,66 @@ const RouteList = ({
           <Text>Bahnhofstrasse 7</Text>
           <Text>6300 Zug</Text>
           <Text>Schweiz</Text>
-          <SpacerV />
-          <View style={AppStyles.containerHorizontal}>
-            <Text>{`${t("model.route.iban")}: ${iban}`}</Text>
-            <IconButton
-              icon="content-copy"
-              onPress={() => ClipboardService.copy(iban)}
-              style={device.SM ? undefined : AppStyles.mla}
-              size={20}
-            />
-          </View>
-          <View style={AppStyles.containerHorizontal}>
-            <Text>{`SWIFT/BIC: ${swift}`}</Text>
-            <IconButton
-              icon="content-copy"
-              onPress={() => ClipboardService.copy(swift)}
-              style={device.SM ? undefined : AppStyles.mla}
-              size={20}
-            />
+
+          <SpacerV height={20} />
+
+          <View style={device.SM && [AppStyles.containerHorizontal, styles.sepaContainer]}>
+            <View style={[styles.sepaItem, device.SM && { flexShrink: 1 }]}>
+              <H4 text={t("model.sepa.instant.title")} style={{ color: Colors.Primary }} />
+              <CopyLine text={`${t("model.route.iban")}: ${instantIban}`} copyText={instantIban} />
+              <CopyLine text={`BIC: ${instantBic}`} copyText={instantBic} />
+              <SpacerV />
+              {[...Array(7).keys()].map((_, i) => (
+                <View key={i} style={[AppStyles.containerHorizontal, { alignItems: "flex-start" }]}>
+                  <View style={i > 3 && { marginLeft: 20 }}>
```

Comment:
Would it make sense to move those two "random" values:
- length of 7
- other style on index greater than 3

to some more describing variables?